include/libtorrent/session_handle.hpp

```diff
@@ -443,6 +443,14 @@ namespace libtorrent
 		void dht_get_peers(sha1_hash const& info_hash);
 		void dht_announce(sha1_hash const& info_hash, int port = 0, int flags = 0);
 
+		// Query the DHT to retrieve a sample of the infohashes that other nodes
+		// currently have in their storage.
+		// The ``target`` is included for iterative lookups so that indexing nodes
+		// can perform a keyspace traversal with a single RPC per node by adjusting
+		// the target value for each RPC. It has no effect on the returned sample value.
+		// The result is posted as a ``dht_sample_infohashes_alert``.
```

Comment:
I would expect the resulting info hashes to have an affinity to be close to ``target`` no? your comment here seem to suggest they are unrelated